akka-remote/src/main/scala/akka/remote/MessageSerializer.scala

```diff
@@ -53,27 +54,28 @@ private[akka] object MessageSerializer {
 
     // FIXME: This should be a FQCN instead
     headerBuilder.serializer = serializer.identifier.toString
```

Comment:
@drewhk I'm not convinced that it's a good idea to use FQCN for the serializer identifier. I think that can become a problem for rolling upgrade scenarios where you want to rename serializer classes.